src/omv/sensor.c

```diff
@@ -525,6 +525,8 @@ int sensor_set_pixformat(pixformat_t pixformat)
         return -1;
     }
 
+    systick_sleep(100); // wait for the camera to settle
```

Comment:
This is usually followed by set_framesize (or the other way around), you'll be adding 200ms delay, and then skip frames is called anyway (or at least 100ms will pass by the time snapshot is called). Is it necessary ?

Reply:
It just hides a corrupt frame that appears when you change modes. I didn't want to put in within the driver as that can be changed via set frame size.